config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -0,0 +1,52 @@
+periodics:
+- interval: 24h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-compatibility-versions
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: compatibility-version-test
+    description: Uses kubetest & kind to run e2e tests from older (n-1..3) kubernetes releases against a latest kubernetes master components w/ --emulated-version=<n-1..3> set.
+    # TODO(aaron-prindle) route the alert email to a rotation vs individual email
+    testgrid-alert-email: aprindle@google.com
+    testgrid-num-columns-recent: '6'
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/krte:latest-master
```

Comment:
@BenTheElder don't we pin the image versions?

```
gcr.io/k8s-staging-test-infra/krte:v20240903-6a352c5344-master
```

Reply:
I used `latest-master` initially as it was used in a periodic kind canary test:

https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml#L66

I have updated this to use no longer use the latest tag but the use the tag you suggested above:
`gcr.io/k8s-staging-test-infra/krte:v20240903-6a352c5344-master`